hal/network/lwip/wiznet/wiznetif.cpp

```diff
@@ -327,7 +327,7 @@ void WizNetif::loop(void* arg) {
         if (p) {
             self->output(p);
         }
-        if (self->inRecv_) {
+        if (self->inRecv_ || self->interrupt_ == PIN_INVALID) {
```

Comment:
Is this working ok? We might need to lower the default timeout from current 500ms for this case I think. 500ms sounds like a lot.

Reply:
This was tested to be working fine. So if the INT pin is not present, let's lower it to 250ms as the compromise.